### PR TITLE
Removes the veteran system

### PR DESCRIPTION
**Pull request by xPokee**
## About The Pull Request
I don't know why it was like that, but people complained

## How This Contributes To The Skyrat Roleplay Experience
honk

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
config: Mine and clown are no longer locked behind veteran
config: Snail isn't veteran only
/:cl:

### CODE REVIEW

**Review by distributivgesetz — APPROVED** [review]

**Comment by distributivgesetz** [comment]
could also remove the `veteran_only` variable

**Comment by xPokee** [comment]
> could also remove the `veteran_only` variable

Want me to do that instead?

**Comment by distributivgesetz** [comment]
yeah

**Review by distributivgesetz — CHANGES_REQUESTED** [review]
Can you either remove this file or remove the field altogether? We want to decommission the veteran system, so we don't need to keep it around.

File: tgui/packages/tgui/interfaces/PreferencesMenu/SpeciesPage.tsx [hunk 1]
```diff
@@ -254,43 +253,7 @@ const SpeciesPageInner = (
...
-                    species.name +
-                    ' - You need to be a veteran to select this race, apply today!';
-                  speciesPage = (
-                    <Tooltip content={tooltipContent}>{speciesPage}</Tooltip>
-                  );
-                }
-                return speciesPage;
-                // SKYRAT EDIT END
-              })}
+              {species.map(([speciesKey, species]) => {})}
```
Comment (distributivgesetz) [inline tgui/packages/tgui/interfaces/PreferencesMenu/SpeciesPage.tsx:256]: This nukes the entirety of the species selection list. Please only remove the veteran early return as well as the tooltip wrapper.

File: code/modules/mob/dead/new_player/new_player.dm [hunk 2]
```diff
@@ -165,8 +165,6 @@
 		return JOB_UNAVAILABLE_LANGUAGE
 	if(job.has_banned_quirk(client.prefs))
 		return JOB_UNAVAILABLE_QUIRK
-	if(job.veteran_only && !SSplayer_ranks.is_veteran(client))
-		return JOB_NOT_VETERAN
```
Comment (distributivgesetz) [inline code/modules/mob/dead/new_player/new_player.dm:169]: Remove this define

File: modular_skyrat/modules/player_ranks/code/player_rank_controller/veteran_controller.dm [hunk 3]
```diff
@@ -1,47 +0,0 @@
...
-
-/datum/player_rank_controller/veteran/get_ckeys_for_legacy_save()
-	if(IsAdminAdvancedProcCall())
-		return
-
-	return GLOB.veteran_list
-
-
-/datum/player_rank_controller/veteran/should_use_legacy_system()
-	return CONFIG_GET(flag/veteran_legacy_system)
```
Comment (distributivgesetz) [inline modular_skyrat/modules/player_ranks/code/player_rank_controller/veteran_controller.dm:40]: Remove these config options from `modular_skyrat\master_files\code\controllers\configuration\entries\skyrat_config_entries.dm`.

**Review by distributivgesetz — CHANGES_REQUESTED** [review]

**Review by distributivgesetz — COMMENTED** [review]
Lgtm, one last thing, remove `#define JOB_NOT_VETERAN` then I'd say this is ready to merge.

File: tgui/packages/tgui/interfaces/PreferencesMenu/SpeciesPage.tsx [hunk 4]
```diff
@@ -1,4 +1,3 @@
-import { classes } from 'common/react';
```
Comment (distributivgesetz) [inline tgui/packages/tgui/interfaces/PreferencesMenu/SpeciesPage.tsx:1]: Reimport this.

**Review by distributivgesetz — APPROVED** [review]

**Pull request merged**